### PR TITLE
stricter comparaison

### PR DESCRIPTION
**Pull request by ArtificialOwl**
- ignore shares to users
- stricter scoring, any non-related share will divide scoring per 10
- ignore low score result

### CODE REVIEW

**Comment by PVince81** [comment]
@ArtificialOwl don't ignore share to users

if two resources X and Y are shared with the set "User1, Circle1, Group1", then they are considered related.
if now we change Y to only be shared with "Circle1, Group1" they are not considered related any more.

user shares also count in the set

**Comment by ArtificialOwl** [comment]
> will stricter comparison completely remove fuzziness or is there a 1% chance that fuzziness still applies in some cases ?

if you share an item to 5 groups, and another item to 4th of them, there are considered related, but will appears down the list. Should I make it stricter ?

**Comment by PVince81** [comment]
@ArtificialOwl yes, please remove any result that is not strictly related.

**Comment by ArtificialOwl** [comment]
> @ArtificialOwl yes, please remove any result that is not strictly related.

done

File: lib/Model/RelatedResource.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ class RelatedResource implements IRelatedResource, IDeserializable, JsonSerializ
 	public static float $IMPROVE_MEDIUM_LINK = 1.3;
 	public static float $IMPROVE_HIGH_LINK = 1.8;
 	public static float $IMPROVE_OCCURRENCE = 1.4;
-	public static float $UNRELATED = 0.1;
+	public static float $UNRELATED = 0.01;
```
Comment (PVince81) [inline lib/Model/RelatedResource.php:52]: would 0 work as well ? 0.01 doesn't sound strict/exact-matching, it's just a reduction of the fuzziness

**Review by PVince81 — APPROVED** [review]
:+1: as discussed, it's a step forward

**Pull request merged**